dashboards/mobile/mobile.json

```diff
@@ -19,11 +19,11 @@
             "id": "viz.markdown"
           },
           "rawConfiguration": {
-            "text": "# Welcome to the New Relic Mobile dashboard! 📱📊\n## Before you start, make sure to follow this little instruction for maximum visibility. After that, you may delete this text box.\n\nThe views in this dashboard are `entity-guid` dependant, so it's important you add it to all your charts' NRQL queries. But fear no more, this is easy as 🍰! Complete these checklists and you'll be ready to go.\n\n## Finding your Entity guid:\n- [ ] Navigate to the [Mobile](https://one.eu.newrelic.com/mobile)  section.\n- [ ] Find your app and click the ⋮ button on the right.\n- [ ] Click on *See metadata & tags* and copy your `entityGuid`.\n\n## Editing your NRQL queries:\n\n- [ ] Go to your charts and click the ⋮ button on the top right corner.\n- [ ] Click on *Edit* and add replace `addYourOwnGuid` with the guid you found. Make sure it remains \"in quotations\".\n\n > NOTE: While some of the charts in this dashboard might work out of the box without this addition, please bear in mind if you don't specify an Entity guid the results displayed will be an aggreagation of *all* your mobile apps in your account."
+            "text": "# Welcome to the New Relic Mobile dashboard! 📱📊\n## Before you start, make sure to follow this little instruction for maximum visibility. After that, you may delete this text box.\n\nYou may notice after installation that charts are showing no data. You can select a specific app to be the default in the variable dropdown labelled `Select Mobile App`:\n- [ ] Click on the ✏️ icon i nthe top right-hand corner of the dashboard\n- [ ] Click on the `Select Mobile App` variable dropdown in the top left-hand corner and then click **Edit**\n- [ ] In the settings sidebar select an app from the `Default value` dropdown\n\nThe dashboard will default to data from this app and you can select any other app from the `Select Mobile App` dropdown.\n\n > NOTE: While some of the charts in this dashboard might work out of the box without an app selected, please bear in mind the results displayed will be an aggreagation of *all* mobile apps in your account.\n \n ---\n \n ⚠️ The dropdown list of apps is currently limited to 10000. If you have more than 10000 apps in your account you can adjust the NRQL query to narrow down results by:\n- [ ] Click on the ✏️ icon inthe top right-hand corner of the dashboard\n- [ ] Click on the `Select Mobile App` variable dropdown and then click **Edit**\n- [ ] Adjust the NRQL query to narrow down results e.g. `WHERE osName = 'iOS'`"
```

Comment:
Please change: "Click on the ✏️ icon i nthe top" -> "Click on the ✏️ icon in the top" which appears twice here

Reply:
thank you kindly